lib/blueprinter/configuration.rb

```diff
@@ -2,7 +2,7 @@
 
 module Blueprinter
   class Configuration
-    attr_accessor :association_default, :datetime_format, :deprecations, :field_default, :generator, :if, :method, :sort_fields_by, :unless, :extractor_default, :default_transformers
+    attr_accessor :association_default, :datetime_format, :deprecations, :field_default, :generator, :if, :method, :sort_fields_by, :unless, :extractor_default, :default_transformers, :custom_array_like_classes
```

Comment:
@ritikesh my solution to the memoization problem was to rename the setting to `custom_array_like_classes` and then make an `array_like_classes` method that is memoized and combines the `custom_array_like_classes` with the standard ones.

This way reads better in my opinion anyway, it's obvious there are some standard array like classes and then you can add custom ones.